### ISSUE TITLE
Reinstate Copyrights via Android Studio

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythtvPlayerForAndroid#151]
@dmfrey DO NOT APPLY THIS YET. I need to force Android Studio to do a commit
that will apply the below (and I'll use your hardcoded name, not $username.)

Reference: https://www.jetbrains.com/help/idea/2016.1/copyright-profiles.html

Add text to Settings->Editor->Copyright->Copyright Profiles. Name it: Default
Enter the following text:
```
$project.name. An application for Android users to play MythTV Recordings and Videos
Copyright (c) $today.year. $username
 
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by 
the Free Software Foundation, either version 3 of the License, or 
(at your option) any later version.
 
This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of 
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
 
You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
In the: 'Allow replacing copyright if old copyright contains' add and Apply:
```
Created by dmfrey on
```
In: Settings->Editor->Copyright, now change: 'No copyright' to 'Default'. Then
click on the + sign and select a scope of Project files.

Now, from the Code->Update Copyright screen, select 'Whole Project', deselect
'Update existing copyrights and ' press OK.

Finally, when pushing commits, make sure the: 'Update copyright' box isn't checked
on the 'Commit Changes' page under the 'Before Commit' section. This assumes
you don't want the year to change from 2015 to 2016, for example. Otherwise,
check it.

**Comment by billmeek** [comment]
@dmfrey, ok, you should be able to apply the steps in the
initial post here to make sure future files get created with
a copyright in them. That actually needs some testing.

**Comment by dmfrey** [comment]
@billmeek Is the original issue up to date?  I see this line:

    In: Settings->Editor->Copyright, now change: 'No copyright' to 'Default'. Then
click on the + sign and select a scope of Project files.

But the previous was to name it 'Default'? Are these just reversed?

**Comment by billmeek** [comment]
I don't think they're reversed.

@dmfrey, not sure what was in yours before. I had reloaded AS some time
ago and had lost all settings. In **Settings->Editor->Copyright->Copyright Profiles**
I had to build a new profile, then above that in **Settings->Editor->Copyright** I
was able to switch from No copyright to the newly created one that I called Default.

Side note, I did create a new file and the new copyright text was automatically
included in it.
